### DISCUSSION TITLE
手机杀死了摄影？

### DISCUSSION THREAD

**Discussion by eason8479** [1101-GS4719-C/Python-Community-GS4719-C#907] [科技新知]
影片連結：https://www.youtube.com/watch?v=QsBXvzbTUJ0

在看完影片後，我就拿起我手中的手機（拿了好幾年了）試了一會兒。
結果影片里提到的功能幾乎都有出現，而且我是用內建的相機，沒額外載app。

不得不說，現在軟硬體的發展真的都是處在歷史的巔峰。
曾經那些花費實驗室電腦算上幾天、甚至幾個月的運算
現在用手機就能輕鬆算到（而且一秒還給你算30+次）

雖說看著手機裡的自己是很好看沒錯。
但是也給人一種不真實的感覺
對於這種撕裂感該如何調適，可能會是伴隨我們這一待人的議題

**Comment by evanchou1204** [comment]
同意現在軟硬體的發展都處在歷史的巔峰這句話，活在這個日新月異的時代，真的感覺一下就會跟不上科技進步了。

**Comment by ching020721** [comment]
感覺現在的科技都沒有太大的突破，維護修改是現在的大宗，創新的發展反而很少

**Comment by Evan0979** [comment]
專業單眼拍出來的還是好很多吧,有在玩攝影的都知道,但現在iphone的相機越來越猛,未來怎麼樣真的不好說

**Comment by CPCpyt** [comment]
我覺得比起技術效果，如何運用才是重點。
再好的相機沒辦法帶給你好的構圖、角度和素材。
